app/assets/javascripts/active_admin/sortable.js.coffee

```diff
@@ -57,6 +57,10 @@ $ ->
         $.ajax
           url: $this.data("sortable-url")
           type: "post"
+          beforeSend: (xhr) -> xhr.setRequestHeader('X-CSRF-Token', $('meta[name="csrf-token"]').attr('content'))
+          headers: {
```

Comment:
Both `headers` and `beforeSend` are needed?

Reply:
@agaidot it's a good question, I'll check with only one attribute